ocs_ci/deployment/hosted_cluster.py

```diff
@@ -197,12 +197,12 @@ def check_odf_prerequisites(self):
         # If hostNetwork is true, then providerAPIServerServiceType is set to NodePort automatically
 
         sc = storage_cluster.get_storage_cluster()
-        sc.check_phase(constants.STATUS_READY)
```

Comment:
If this is removed, what will be the alternative for status check ?

Reply:
this was a new check added in another PR days earlier, added with more changes. I did not have chance to test it due to lack of env and converged build. So running it I saw syntax issue, and removed it. Most important deployment now passes. Later I will improve this function with more checks.